wpimath/src/main/native/cpp/geometry/Pose3d.cpp

```diff
@@ -116,22 +116,23 @@ Twist3d Pose3d::Log(const Pose3d& end) const {
   // Implementation from Section 3.2 of https://ethaneade.org/lie.pdf
   const auto transform = end.RelativeTo(*this);
 
-  auto u = Vectord<3>{transform.X().value(), transform.Y().value(),
-                      transform.Z().value()};
-  auto rvec = transform.Rotation().GetQuaternion().ToRotationVector();
+  Eigen::Vector3d u{transform.X().value(), transform.Y().value(),
+                    transform.Z().value()};
+  Eigen::Vector3d rvec =
+      transform.Rotation().GetQuaternion().ToRotationVector();
 
-  auto omega = RotationVectorToMatrix(rvec);
-  auto omegaSq = omega * omega;
-  auto theta = rvec.norm();
-  auto thetaSq = theta * theta;
+  Eigen::Matrix3d omega = RotationVectorToMatrix(rvec);
+  Eigen::Matrix3d omegaSq = omega * omega;
+  double theta = rvec.norm();
+  double thetaSq = theta * theta;
 
   double C;
-  if (theta < 1E-9) {
+  if (std::abs(theta) < 1E-9) {
     // Taylor Expansions around θ = 0
     // A = 1/1! - θ²/3! + θ⁴/5!
     // B = 1/2! - θ²/4! + θ⁴/6!
     // C = 1/6 * (1/2 + θ²/5! + θ⁴/7!)
```

Comment:
This comment isn't an equivalent equation to the code.

Reply:
I did a double check with wolframalpha to make sure I got the taylor expansions right - they were wrong again.

Pose3d::exp - https://www.wolframalpha.com/input?i=series+expansion%281-sin%28x%29%2Fx%29%2Fx%5E2+at+x%3D0

Pose3d::log - https://www.wolframalpha.com/input?i=series+expansion+%281-%28sin%28x%29%2Fx%29%2F%282*%281-cos%28x%29%29%2Fx%5E2%29%29%2Fx%5E2+at+x%3D0

Fixing shortly.

Reply:
In that case, we should include the wolfram alpha links in the comment as a source.